include/logging/log_core.h

```diff
@@ -14,6 +14,40 @@
 extern "C" {
 #endif
 
+/* MOJ KOD */
+
+#include <zephyr.h>
+#include <device.h>
+#include <gpio.h>
+#include <hal/nrf_gpio.h>
+
+#if (CONFIG_LOG_FRONTEND_PORT_NUMBER == 0)
+#define LOG_FRONTEND_GPIO_REG_PTR NRF_P0
+#elif (CONFIG_LOG_FRONTEND_PORT_NUMBER == 1)
+#define LOG_FRONTEND_GPIO_REG_PTR NRF_P1
+#endif
+
+struct LOG_FRONTEND_PIN_NUMBERS
+{
+	uint8_t BIT0;
+	uint8_t BIT1;
+	uint8_t BIT2;
+	uint8_t BIT3;
+	uint8_t BIT4;
+	uint8_t BIT5;
+	uint8_t BIT6;
+	uint8_t BIT7;
+	uint8_t CLK;
+};
+
+uint32_t create_mask_gpio(struct LOG_FRONTEND_PIN_NUMBERS * pin_nums, uint8_t message);
```

Comment:
in Zephyr types are defined in shorter form, please use `u32_t`, `u8_t` and so on.